packages/core/src/features/gdcapi/filters.ts

```diff
@@ -260,11 +260,13 @@ export interface GqlExcludeIfAny {
 export interface GqlIntersection {
   readonly op: "and";
   readonly content: ReadonlyArray<GqlOperation>;
+  readonly isLoggedIn?: boolean;
```

Comment:
Is this actually getting used? Why do we need this state both on the cohort and in the filters?